imap_processing/ena_maps/ena_maps.py

```diff
@@ -425,7 +429,8 @@ def __init__(
     def project_pset_values_to_map(
         self,
         pointing_set: PointingSet,
-        value_keys: list[str] | None = None,
+        pset_value_keys: list[str] | None = None,
+        skymap_value_keys: list[str] | None = None,
```

Comment:
Do we need to allow for the renaming here too? I feel like this is another extra argument that could lead to confusion and someone can later just take their map and rename variables instead. It also requires things to be in the same order to be mapped properly leading to another potential gotcha.

Reply:
I added `skymap_value_keys` for optional renaming because we may need to push _and_ pull some quantities. E.g., we may need "counts_pushed" and "counts_pulled". I could have a suffix instead, but I lean towards the user making a more explicit choice of names.

Reply:
> we may need

I would leave it until we need it then. I would say someone should make two separate maps then and combine/work on them later.

Reply:
Makes sense - I think I often add this sort of early over-flexibility, which I can see affects legibility and potentially maintainability. I'm working to reduce it!

Reply:
Dropped this parameter.